sphinx-doc/howto/prevent-particles-from-moving.rst

```diff
@@ -0,0 +1,28 @@
+.. Copyright (c) 2009-2023 The Regents of the University of Michigan.
+.. Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+How to prevent particles from moving
+====================================
+
+MD simulations
+--------------
+
+To prevent a subset of particles from moving in MD simulations omit the stationary particles from
```

Comment:
```suggestion
To prevent a subset of particles from moving in MD simulations, omit the stationary particles from
```

Reply:
I reworded this in a different way to shorten and avoid the comma.